.buildscript/deploy_website.sh

```diff
@@ -1,41 +0,0 @@
-#!/bin/bash
```

Comment:
`./gradlew dokka && mkdocs gh-deploy` are enough to publish docs, no need to clone the repo.